samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -59,7 +59,14 @@ class InferReqWrap final {
         _startTime = Time::now();
         _request.infer();
         _endTime = Time::now();
-        _callbackQueue(_id, _lat_group_id, get_execution_time_in_milliseconds());
+        double _execTime = getExecutionTimeInMilliseconds();
```

Comment:
It seems that wrong function `getExecutionTimeInMilliseconds` is used instead of `get_execution_time_in_milliseconds`
```
error: ‘getExecutionTimeInMilliseconds’ was not declared in this scope; did you mean ‘get_execution_time_in_milliseconds’?
   62 |         double _execTime = getExecutionTimeInMilliseconds();
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                            get_execution_time_in_milliseconds
```